providers/amazon/src/airflow/providers/amazon/aws/operators/glue.py

```diff
@@ -254,7 +265,7 @@ def execute_complete(self, context: Context, event: dict[str, Any] | None = None
 
         if validated_event["status"] != "success":
             raise AirflowException(f"Error in glue job: {validated_event}")
-        return validated_event["value"]
+        return validated_event["run_id"]
```

Comment:
Why did we make this change? What are the implications of changing the return value? Is this not a breaking change?